src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -1088,6 +1088,29 @@ bool WholeBodyDynamicsDevice::readFTSensors(bool verbose)
             yWarning() << "wholeBodyDynamics warning : FT sensor " << sensorName << " was not readed correctly, using old measurement";
         }
 
+        if (ftMeasurement.size() != 6)
```

Comment:
This needs to be removed because wholeBodyDynamics can be used with robots that have an arbitrary number of FT sensors.